source/menus/quest_status_menu.cpp

```diff
@@ -281,6 +353,62 @@ void QuestStatusMenu::draw() {
         dComIfGs_setPowerBracelets(new_power_bracelets_item_id);
         dComIfGs_setPowerBraceletsOwned(powerBraceletsIdToPowerBraceletsOwned(new_power_bracelets_item_id));
         break;
+    case MENU_ITEM_QUIVER:
+        new_arrows_capacity = dComIfGs_getArrowCapacity();
+        Cursor::moveListSimple(new_arrows_capacity);
+         if (new_arrows_capacity == NO_QUIVER - 1) {
+            new_arrows_capacity = NO_QUIVER;
+        } else if (new_arrows_capacity == NO_QUIVER + 1) {
+              new_arrows_capacity = ARROWS_30;
+        } else if (new_arrows_capacity == NO_QUIVER) {
+            new_arrows_capacity = NO_QUIVER;

```

Comment:
This "else if" is redundant: if new_arrows_capacity is equal to NO_QUIVER, then set new_arrows_capacity equal to NO_QUIVER.

new_arrows_capacity is already equal to NO_QUIVER